src/libs/DateUtils.js

```diff
@@ -45,12 +47,12 @@ Onyx.connect({
  * @private
  */
 function getLocalMomentFromDatetime(locale, datetime, currentSelectedTimezone = timezone.selected) {
-    moment.locale(locale);
     if (!datetime) {
-        return moment.tz(currentSelectedTimezone);
+        const currentDate = new Date();
+        return formatInTimeZone(currentDate, currentSelectedTimezone, "yyyy-MM-dd'T'HH:mm:ssXXX");
     }
-
-    return moment.utc(datetime).tz(currentSelectedTimezone);
+    const date = new Date(datetime);
+    return formatInTimeZone(date, currentSelectedTimezone, "yyyy-MM-dd'T'HH:mm:ssXXX");
```

Comment:
I guess those formats should be kind of constants.

Reply:
agree, needs to be added

Reply:
Also, is it 100% necessary to use `new Date` before? In the doc I see it allows to get strings, numbers, etc.